<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export to ...</string>
     </property>
     <addaction name="actionHtml"/>
    </widget>
    <addaction name="action_Populate"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionprint"/>
    <addaction name="actionPrint_Preview"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Populate"/>
   <addaction name="actionClear_Shifts"/>
   <addaction name="separator"/>
   <addaction name="actionprint"/>
   <addaction name="actionPrint_Preview"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Populate">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/populate</normaloff>:/icons/populate</iconset>
   </property>
   <property name="text">
    <string>&amp;Populate</string>
   </property>
   <property name="toolTip">
    <string>Fill's table with initial values</string>
   </property>
  </action>
  <action name="actionClear_Shifts">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit_clear</normaloff>:/icons/edit_clear</iconset>
   </property>
   <property name="text">
    <string>Clear Shifts</string>
   </property>
  </action>
  <action name="actionprint">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-print</normaloff>:/icons/document-print</iconset>
   </property>
   <property name="text">
    <string>print</string>
   </property>
   <property name="toolTip">
    <string>Print's the current shifts table</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-print-preview</normaloff>:/icons/document-print-preview</iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
   <property name="toolTip">
    <string>Show how printed table will look like.</string>
   </property>
  </action>
  <action name="actionHtml">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/export</normaloff>:/icons/export</iconset>
   </property>
   <property name="text">
    <string>Html</string>
   </property>
   <property name="toolTip">
    <string>Export current table to Html document</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
